### ISSUE TITLE
Target_value should to be in temp units

### ISSUE THREAD

**Issue by msp1974** [msp1974/homeassistant-jlrincontrol#30]
InControl values are between 31 and 57 which does not represent actual temp value.  Amend so that it converts from temp values to these required values.

**Comment by zachamato** [comment]
I am actually messing around with this now but pretty novice when it comes to this level of python (Im usually just writing simple scripts and modules for my job). 

To get the real temperature of the target value you either: 
+27 for Fahrenheit 
or 
/2 for Celsius then round to the nearest fifth.

I created a sensor for this, and it might make sense to have a HA service to set the target value. I also started to look at creating all of this in a climate.yaml file so the remote start would be represented as a climate device (similar to this: https://github.com/robinostlund/volkswagencarnet).



(Edit: adding for reference https://github.com/g1zm0e/JLRICRCommon-Redev/blob/master/feature_documentation/TargetTemperature.md )
